inc/saihostintf.h

```diff
@@ -230,6 +230,18 @@ typedef enum _sai_hostif_trap_id_t
     /** default packet action is forward */
     SAI_HOSTIF_TRAP_ID_MLD_V2_REPORT = 0x0000200f,
 
+    /** IP packets to local router IP address (routes with SAI_ROUTE_ATTR_NEXT_HOP_ID=SAI_SWITCH_ATTR_CPU_PORT)
```

Comment:
The idea is to use the new trap SAI_HOSTIF_TRAP_ID_IP2ME
Classify it to a trap group, attach a policer to the trap group (SAI_HOSTIF_TRAP_GROUP_ATTR_POLICER) and define the queue (SAI_HOSTIF_TRAP_GROUP_ATTR_QUEUE).
Internally, each vendor can implement with his SDK host IFC API, or with ACL API.
The traffic that matches this new trap, has next hop id = cpu port, as explained in
https://github.com/opencomputeproject/SAI/pull/106